src/Allocation.jl

```diff
@@ -64,19 +61,22 @@ function run_timestep(c::Allocation, tt::Int)
         fips = draws[pp, :fips] < 10000 ? "0$(draws[pp, :fips])" : "$(draws[pp, :fips])"
         rr = findfirst(mastercounties[:fips] .== fips)
         if rr > 0
-            v.swsupply[rr, tt] += p.withdrawals[pp, tt]
-            v.swreturn[rr, tt] += p.returns[pp, tt]
+            v.swsupply[rr, tt] += p.swwithdrawals[pp, tt]
+        
+	    v.swreturn[rr, tt] += p.returns[pp, tt]
         end
-        v.copy_withdrawals[pp, tt] = p.withdrawals[pp, tt]
+        v.copy_swwithdrawals[pp, tt] = p.swwithdrawals[pp, tt]
         v.copy_returns[pp, tt] = p.returns[pp, tt]
     end
 
     for rr in d.regions
-        v.watergw[rr,tt] = p.waterfromgw[rr,tt]
-        v.waterallocated[rr,tt] = p.waterfromgw[rr,tt]+p.waterfromsupersource[rr,tt] + v.swsupply[rr, tt]
-        v.cost[rr, tt] = p.waterfromgw[rr,tt]*p.costfromgw[rr,tt] + v.swsupply[rr,tt]*p.costfromsw[rr,tt] + p.waterfromsupersource[rr,tt]*p.costfromsupersource
+        aa = rr
```

Comment:
Not essential, but that was in case aquifers do not match regions. We could have more than one aquifer in a given region (which might be the case soon with a surficial and deep aquifers) or several region in the same aquifer ...